src/app/client/src/app/modules/dashboard/components/report/report.component.ts

```diff
@@ -155,8 +155,9 @@ export class ReportComponent implements OnInit {
               map((apiResponse) => {
                 const [data, reportSummary] = apiResponse;
                 const result: any = Object.assign({});
-                const chart = (charts && this.reportService.prepareChartData(charts, data, updatedDataSource,
+                let chart = (charts && this.reportService.prepareChartData(charts, data, updatedDataSource,
```

Comment:
move this logic inside prepareChartData method only..

Reply:
resolved